torch/onnx/__init__.py

```diff
@@ -75,10 +75,41 @@ def export(model, args, f, export_params=True, verbose=False, training=TrainingM
         export_raw_ir (bool, default False): [DEPRECATED. use operator_export_type]
             export the internal IR directly instead of converting it to ONNX ops.
         operator_export_type (enum, default OperatorExportTypes.ONNX):
-            OperatorExportTypes.ONNX: all ops are exported as regular ONNX ops.
-            OperatorExportTypes.ONNX_ATEN: all ops are exported as ATen ops.
-            OperatorExportTypes.ONNX_ATEN_FALLBACK: if symbolic is missing, fall back on ATen op.
-            OperatorExportTypes.RAW: export raw ir.
+            OperatorExportTypes.ONNX: All ops are exported as regular ONNX ops
+            (with ONNX namespace).
+            OperatorExportTypes.ONNX_ATEN: All ops are exported as ATen ops
+            (with aten namespace).
+            OperatorExportTypes.ONNX_ATEN_FALLBACK: If an ATen op is not supported
+            in ONNX or its symbolic is missing, fall back on ATen op. Registered ops
+            are exported to ONNX regularly.
+            Example graph:
+                graph(%0 : Float):
+                  %3 : int = prim::Constant[value=0]()
+                  %4 : Float = aten::triu(%0, %3) # missing op
+                  %5 : Float = aten::mul(%4, %0) # registered op
+                  return (%5)
+            is exported as:
+                graph(%0 : Float):
+                  %1 : Long() = onnx::Constant[value={0}]()
+                  %2 : Float = aten::ATen[operator="triu"](%0, %1)  # missing op
+                  %3 : Float = onnx::Mul(%2, %0) # registered op
+                  return (%3)
+            OperatorExportTypes.RAW: Export raw ir.
+            OperatorExportTypes.ONNX_FALLTHROUGH: If an op is not supported in
+            ONNX, fall through and export the operator as is. Using this mode,
+            a custom operator can be registered and implemented by the user for
```

Comment:
@BowenBao The description here is updated. Please take a look and let me know if you have a better suggestion. Thanks.

Reply:
I think i mis-understood the `custom op` definition in the description previously. 

"If an op is not exportable to ONNX, fall through and export the operator as is, as a custom ONNX op. Using this mode, it can be registered and implemented by the user for their runtime backend."